Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -301,6 +301,13 @@ type AndroidProps = $ReadOnly<{|
    * @platform android
    */
   overScrollMode?: ?('auto' | 'always' | 'never'),
+  /**
+   * Causes the scrollbars not to turn transparent when they are not in use.
+   * The default value is false.
+   *
+   * @platform android
+   */
+  persistentScrollbar?: ?boolean,
```

Comment:
O guess we can be more strict about this type and just go with `boolean` if it's set

Reply:
Yeah, I think this is a good suggestion.

Reply:
All the prop types are defined like this. I'd rather keep it consistent than strict, and I assume there might be a reason for this. @grabbou Do you know who might shed some light on why these types are defined as optional on both sides?

Reply:
Then I'd leave it as is. I think it might be for historical reasons. @TheSavior do you happen to know the answer to this?